mitodl/ocw-hugo-themes
# Update OCW logos and improve design alignments

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/3385

### Description (What does it do?)
Updates OCW Logos and aligns the designs consistently.
Eg.
Header, footer, and search bar on homepage are now aligned.
Although not everything is aligned the same across both themes, but things (mostly header, footer, their `max-width`s) are now more consistent within the themes.

### How can this be tested?
1. Follow the discussions on the ticket / Look at figma designs from there
2. Look at different kinds of pages, different themes, apart from logos. To ensure everything is working correctly (because this isn't only a logo PR)
3. Make sure to check for different screen sizes.

For www online theme:
`yarn start www`

For www offline theme:
`yarn build ~/path/ocw-content-rc/ocw-www/ ~/path/ocw-hugo-projects/ocw-www/config-offline.yaml`
(then go to ocw-www content directory, and`dist/index.html`)

For course theme:
`yarn start course`

For offline course theme:
`yarn build ~/path/ocw-content-rc/9.40-spring-2018/ ~/path/ocw-hugo-projects/ocw-course-v2/config-offline.yaml`
(Then go to course content directory, and `dist/index.html`
Although do test on a different course than this one

Note:
You will see:
```
      <img
        src="/static_shared/images/mit_ol.svg"
        width="140px"
        alt="MIT Open Learning"
      />
```
in `Footer.tsx`
Webpack currently creates hashed version of every static image asset. This means, `mit_ol.svg` will not exist. And here, in a `.tsx` file, we cannot get our mapping too. The only reason why our [Search page](https://ocw.mit.edu/search/?q=) footer logo is currently working is because the image already existed from a pre-hash build.

Any way, as a fix to this logo ticket, what I have done is uploaded the `mit_ol.svg` logo to our AWS buckets (all OCW content buckets for RC and Prod). We can work on this problem in another ticket (probably would need a webpack rule I assume for its fix).
So we have the logo available here:
https://ocw.mit.edu/static_shared/images/mit_ol.svg
https://live-qa.ocw.mit.edu/static_shared/images/mit_ol.svg
So it should work in other environments.

To test locally, just place the `mit_ol.svg` in `static_shared` directory with the same name, it should work.

### Screenshots (if appropriate):
### WWW homepage:
<img width="1361" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/f7356542-046f-4c4b-86ca-123442043198">
<img width="1378" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/0a4792d2-5108-4307-8f53-bf888f49aa9b">


### Search page
<img width="1376" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/d0866f04-29ba-462c-8caa-a0918e98fee0">


### Course page
<img width="1431" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/71f1578c-fd07-4ccb-bd46-5b4580f972c9">
<img width="1440" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/8950c873-ef06-4958-a65a-80dbbefca710">




### Additional Context
